2025-09-18 13:55:44.683894-03:00 jdbc[34]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'CERVEJAS' (TB_OBJECTS: ""CATEGORY"" TINYINT)"; SQL statement:
UPDATE "PUBLIC"."TB_OBJECTS" SET "ID"=? ,"CATEGORY"=? ,"DESCRIPTION"=? ,"IMAGE"=? ,"NAME"=? ,"PRICE"=? ,"QUANTITY"=? ,"VALIDITY"=?  WHERE "ID"=? [22018-232]
2025-09-18 13:55:44.686883-03:00 jdbc[34]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'CERVEJAS' (TB_OBJECTS: ""CATEGORY"" TINYINT)"; SQL statement:
UPDATE "PUBLIC"."TB_OBJECTS" SET "ID"=? ,"CATEGORY"=? ,"DESCRIPTION"=? ,"IMAGE"=? ,"NAME"=? ,"PRICE"=? ,"QUANTITY"=? ,"VALIDITY"=?  WHERE "ID"=? [22018-232]
2025-09-18 13:58:24.447479-03:00 jdbc[34]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'CERVEJAS' (TB_OBJECTS: ""CATEGORY"" TINYINT)"; SQL statement:
UPDATE "PUBLIC"."TB_OBJECTS" SET "ID"=? ,"CATEGORY"=? ,"DESCRIPTION"=? ,"IMAGE"=? ,"NAME"=? ,"PRICE"=? ,"QUANTITY"=? ,"VALIDITY"=?  WHERE "ID"=? [22018-232]
2025-09-18 13:58:24.447479-03:00 jdbc[34]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'CERVEJAS' (TB_OBJECTS: ""CATEGORY"" TINYINT)"; SQL statement:
UPDATE "PUBLIC"."TB_OBJECTS" SET "ID"=? ,"CATEGORY"=? ,"DESCRIPTION"=? ,"IMAGE"=? ,"NAME"=? ,"PRICE"=? ,"QUANTITY"=? ,"VALIDITY"=?  WHERE "ID"=? [22018-232]
2025-09-18 14:12:29.192903-03:00 jdbc[35]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-232]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:217)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:134)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:144)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:370)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:207)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:488)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:752)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:731)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)
	at java.base/java.lang.Thread.run(Thread.java:1447)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 16 more
